Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
```

Comment:
Not about this line, but should this be merged into `next` instead?

Reply:
I don't think so -- there's nothing consensus breaking about this PR, so it doesn't need to wait until Stacks 2.1.